lib/dradis/plugins/projects/export/v1/template.rb

```diff
@@ -41,7 +41,7 @@ def build_evidence_for_node(builder, node)
             evidence_builder.author(evidence.author)
             evidence_builder.tag!('issue-id', evidence.issue_id)
             evidence_builder.content do
-              evidence_builder.cdata!(evidence.content)
+              cdata!(evidence_builder, evidence.content || "")
```

Comment:
@dormi the PR with the pipe(`||`) change has been closed. See: https://github.com/dradis/dradis-projects/pull/7.